src/main/kotlin/app/revanced/patches/youtube/layout/castbutton/annotations/CastPatchCompatibility.kt

```diff
@@ -0,0 +1,13 @@
+package app.revanced.patches.youtube.layout.castbutton.annotations
+
+import app.revanced.patcher.annotation.Compatibility
+import app.revanced.patcher.annotation.Package
+
+@Compatibility(
+    [Package(
+        "com.google.android.youtube", arrayOf("17.14.35", "17.17.34", "17.19.36", "17.20.37", "17.22.36") // only 17.20.37 is tested

```

Comment:
```suggestion
        "com.google.android.youtube", arrayOf("17.20.37")
```
If only `17.20.37` is tested, you should only put that in the compatibility list.
Only when a patch is tested for a certain version, may it be added to the compatibility list.